algorithms/ar-bsrh/python2/binary_search.py

```diff
@@ -0,0 +1,27 @@
+def binarysearch(arr,x):
+  low=0
+  high=len(arr)-1
+  mid=0
+  while(low<=high): 
```

Comment:
Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6277716)

Reply:
this is happening when you have added unwanted blank spaces at the end of the code lines. just remove them and the issue will be gone.